jest/config.js

```diff
@@ -10,4 +10,7 @@ module.exports = {
     'node_modules',
     'node_modules/(?!(@react-native-community|react-native)/)',
   ],
+  transformIgnorePatterns: [
```

Comment:
@iocedano  DO you think this got any implications? We solved the failing test like this